torchrl/envs/libs/brax.py

```diff
@@ -0,0 +1,354 @@
+from typing import Dict, Optional, Union
+
+import torch
+from tensordict.tensordict import TensorDict, TensorDictBase
+
+from torchrl.data import (
+    CompositeSpec,
+    NdBoundedTensorSpec,
+    NdUnboundedContinuousTensorSpec,
+)
+from torchrl.envs.common import _EnvWrapper
+from torchrl.envs.libs.jumanji import _torchrl_data_to_spec_transform
```

Comment:
we should definitely not import stuff from lib to lib.
Here I would put `_torchrl_data_to_spec_transform` in a utils file, jax_utils (and rename it to a less generic name).

Reply:
done